OoyalaSkinSampleApp/build.gradle

```diff
@@ -1,18 +1,64 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    // Define versions in a single place
```

Comment:
What do you mean by "Define versions in a single place"? Are you going to move all these versions to build.gradle of the root project later?

Reply:
They are already defined in the root project for each app, as you can see below. This way when we work with core, all dependencies will have the same version and we will not have to manually force a desired version.

Reply:
I suggested moving versions to the root of all sample apps to avoid duplication.